recipes/acryl-datahub/meta.yaml

```diff
@@ -0,0 +1,76 @@
+{% set name = "acryl-datahub" %}
+{% set version = "0.15.0.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.org/packages/source/{{ name[0] }}/{{ name }}/acryl_datahub-{{ version }}.tar.gz
+  sha256: 6b1b8591bb485b2aa51010052758e9a7b2dee2030c8b373b72e944c570935f28
+
+build:
```

Comment:
This looks like this package should probably be `noarch: python`. See:
https://conda-forge.org/docs/maintainer/knowledge_base/#noarch-python

Take note of changes related to `python_min`. I think the main thing you were struggling with is that even though upstream says it supports `python>=3.8`, in reality that metadata is wrong since they are using typing syntax that would require you to set `python_min = 3.10`

Reply:
There are tests running that check that it works on Python 3.8 https://github.com/datahub-project/datahub/actions/runs/12946781664

On the pipelines it was running it on Python 3.12